frontend/src/__tests__/cypress/cypress/pages/pipelines/pipelineVersionImportModal.ts

```diff
@@ -38,6 +38,22 @@ class PipelineImportModal extends Modal {
     this.findUploadPipelineInput().selectFile([filePath], { force: true });
   }
 
+  findUploadPipelineRadio() {
+    return this.find().findByTestId('upload-file-radio');
+  }
+
+  findImportPipelineRadio() {
+    return this.find().findByTestId('import-url-radio');
+  }
+
+  findPipelineUrlInput() {
+    return this.find().findByTestId('pipeline-url-input');
+  }
+
+  findCodeSourceInput() {
```

Comment:
This doesn't seem to be used anywhere, which leads to the question of whether tests should exist for the "code source" input. I think have at least 1 integration test using this would make sense since the ticket states "Support source code upload".

Reply:
There is no test becuase there is no UI to display the source code. I mean i could make sure that the request body has it set